src/platform/user/profile/vap-svc/constants/index.js

```diff
@@ -16,6 +18,14 @@ export const ADDRESS_TYPES = {
   OVERSEAS_MILITARY: 'OVERSEAS MILITARY',
 };
 
+// TODO: Merge with ADDRESS_TYPES above, or replace them both with the
+// ADDRESS_TYPES constant in platform/forms/address/helpers.js
+export const ADDRESS_TYPES_ALTERNATE = Object.freeze({
```

Comment:
Eww. But, yeah, dealing with these two different `ADDRESS_TYPE` objects shouldn't be part of your work here.

Reply:
That was my thinking, had to limit the scope on this PR and not delve into that can of worms. But it would be great to clean these up at some point. Thanks for the review!